src/output.cpp

```diff
@@ -85,6 +85,25 @@ std::vector<std::string> split_by_color(const std::string &s)
     return ret;
 }
 
+std::string remove_color_tags(const std::string &s)
```

Comment:
If you're going to return a new string anyway, why not take the param by value and use it?